docker/.env

```diff
@@ -56,6 +56,7 @@ MF_USERS_ADMIN_EMAIL=admin@example.com
 MF_USERS_ADMIN_PASSWORD=12345678
 MF_USERS_RESET_PWD_TEMPLATE=users.tmpl
 MF_USERS_PASS_REGEX=^.{8,}$
+MF_ONLY_ADMIN_CREATES_USER=
```

Comment:
Is `MF_ONLY_ADMIN_CREATES_USER` a good name for ENV var? Would it better to call it `MF_SELF_SIGNON`, and if this one is false, then only admin can create users. Or am I missing something?

Reply:
`MF_SELF_SIGNON` is okay for me. I'll rename the environment variable.

By default, everybody can create a new user. If `MF_ONLY_ADMIN_CREATES_USER` is empty, everybody is allowed to create a new user. Otherwise (`MF_ONLY_ADMIN_CREATES_USER` is not empty, i.e., an arbitrary string), only the admin is allowed to create a new user.

Reply:
Ah, then this is not the same as self-sign-on. OK, we can keep this one.

Reply:
I still don't like this naming, but let's fix it in the next PR.

Reply:
Yes, @buraksekili please do not forget to fix one in the next PR